packages/buttons/src/elements/Anchor.tsx

```diff
@@ -10,12 +10,11 @@ import PropTypes from 'prop-types';
 import { StyledAnchor, StyledExternalIcon } from '../styled';
 
 export interface IAnchorProps extends AnchorHTMLAttributes<HTMLAnchorElement> {
-  /** Apply danger styling */
+  /** Determines if danger styling is used */
   isDanger?: boolean;
   /**
-   * Used when the anchor navigates to an external resource. Applies `target="_blank"`
-   * along with `rel="noopener noreferrer"` to ensure safe
-   * [cross-origin destination links](https://web.dev/external-anchors-use-rel-noopener/).
+   * Determines if the anchor navigates to an external resource.
+   * Applies `target="_blank"` and `rel="noopener noreferrer"` to ensure safe [cross-origin destination links](https://web.dev/external-anchors-use-rel-noopener/).
```

Comment:
Formatting tweak... These sentences should be connected as a paragraph, similar to the former description. We like to prevent long lines – please split around col 80-100, when possible.

The second sentence feels like it could use some work. Something like:

> When enabled, `target="_blank"` and `rel="noopener noreferrer"` attributes are added to ensure...

Reply:
Fixed in latest update